OpenRA.Platforms.Default/MultiTapDetection.cs

```diff
@@ -31,12 +31,12 @@ public static int InfoFromMouse(byte button)
 			return clickHistoryCache[button].LastTapCount();
 		}
 
-		public static int DetectFromKeyboard(Keycode key)
+		public static int DetectFromKeyboard(Pair<Keycode, Modifiers> key)
```

Comment:
IMO it would be cleaner if this took in two arguments rather than a `Pair`.

Reply:
Sure, fixed